Upstream: No, upstream added additional make target after stable tag
Reason: gitstatus Makefile downloads binary instead of building one and even when it's present - downloads new one

--- a/gitstatus/Makefile
+++ b/gitstatus/Makefile
@@ -35,7 +35,7 @@ clean:
 	rm -rf -- $(OBJDIR)
 
 pkg:
-	GITSTATUS_DAEMON= GITSTATUS_CACHE_DIR=$(shell pwd)/usrbin ./install -f
+	GITSTATUS_DAEMON= GITSTATUS_CACHE_DIR=$(shell pwd)/usrbin ./install
 	$(or $(ZSH),:) -fc 'for f in *.zsh install; do zcompile -R -- $$f.zwc $$f || exit; done'
 
 -include $(OBJS:.o=.dep)
